src/view/navigation/index.tsx

```diff
@@ -93,28 +109,35 @@ class AppNavigation extends React.Component<Props, {}> {
         break;
 
       default:
-        throw new Error('should not be default case');
+        throw new Error('No such page type');
```

Comment:
maybe include the name of the type in the message for debugging purposes?

Reply:
good call :+1: